appveyor.yml

```diff
@@ -5,9 +5,8 @@ environment:
   - TARGET: x86_64-pc-windows-msvc
     ALLOW_PR: 1
   - TARGET: i686-pc-windows-msvc
```

Comment:
The MSI version is nowhere near finished. I want to enable it on CI just so it won't break. Maybe it would make sense to also set `ALLOW_PR: 1` for it.